Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ.yml

```diff
@@ -81,6 +75,14 @@ script:
       name: active
     - description: Numeric value of baseline risk score, users above this will be returned.
       name: risk
+    - auto: PREDEFINED
+      description: use custom filter to find user in IIQ
```

Comment:
Please write here a note that if this argument is used then the other arguments (email,active,risk) will not be taken into account in the search filter.